ethereum/eth/src/test/java/tech/pegasys/pantheon/ethereum/eth/manager/EthServerTest.java

```diff
@@ -45,7 +46,11 @@
 
   @Before
   public void setUp() {
-    new EthServer(blockchain, worldStateArchive, ethMessages, 2, 2, 2, 2);
+    new EthServer(
+        blockchain,
+        worldStateArchive,
+        ethMessages,
+        new EthereumWireProtocolConfiguration(2, 2, 2, 2));
```

Comment:
Can we make this constructor private so that modifications must go through the builder?  If we stick with the builder pattern, it will save us pain in the future.